tensorflow.md

```diff
@@ -1,32 +1,65 @@
-# Building TensorFlow on Graviton2
+# ML inference on Graviton CPUs with TensorFlow
 
-Install bazel build system:
+There are two ways to use tensforflow based ml inferencing on Graviton CPUs.
+- Tensorflow DLAMI
+- Tensorflow Docker container
+
+**Using AWS DLAMI:**
+Launch Graviton instance with the DLAMI: Deep Learning AMI Graviton GPU TensorFlow 2.7.0 (Ubuntu 20.04) 20211122. Now, the platform is ready for start building Tensorflow based ML inference applications. The DLAMI comes with the additional libraries preinstalled, eg: rust compilers, transformers etc, to enable transformers based models (eg: bert) inferencing.
```

Comment:
It would more useful to explain how to find this AMI on the AWS console as versions will change i.e: "To launch a Graviton instance with a DLAMI via the EC2 console:
1. Go to Launch instances
2. Click on "Community AMIs"
2. Filter AMIs by clicking Ubuntu and Arm (64-bit) check boxes
3. Search for "Deep Learning" in search field to list the current Ubuntu based DLAMIs"

Providing a small AWS CLI snippet to find the AMIs would also be great.